### PR TITLE
Disable analytics

### PR DESCRIPTION
**Pull request by keliramu**

### CODE REVIEW

File: daemon/vpn/nordlynx/libtelio/libtelio.go [hunk 1]
```diff
@@ -143,35 +143,29 @@ type persistentKeepAliveConfig struct {
...
-	telioConfig.Lana.EventPath = eventPath
-	telioConfig.Lana.Prod = prod
-	telioConfig.Nurse.Fingerprint = deviceID
+	if telioConfig.Lana != nil {
+		telioConfig.Lana.EventPath = eventPath
+		telioConfig.Lana.Prod = prod
+		if telioConfig.Nurse != nil { // nurse depends on lana
+			telioConfig.Nurse.Fingerprint = deviceID
+		} else {
+			telioConfig.Nurse = nil
```
Comment (bartoszWojciechO) [inline daemon/vpn/nordlynx/libtelio/libtelio.go:163]: Wouldn't `telioConfig.Nurse` already be nil in this branch?

File: daemon/vpn/nordlynx/libtelio/libtelio.go [hunk 2]
```diff
@@ -143,35 +143,29 @@ type persistentKeepAliveConfig struct {
...
+			return nil, fmt.Errorf("unmarshaling telio remote config json string: %s\n", err)
 		}
 	}
-	telioConfig.Lana.EventPath = eventPath
-	telioConfig.Lana.Prod = prod
-	telioConfig.Nurse.Fingerprint = deviceID
+	if telioConfig.Lana != nil {
+		telioConfig.Lana.EventPath = eventPath
+		telioConfig.Lana.Prod = prod
+		if telioConfig.Nurse != nil { // nurse depends on lana
```
Comment (EmilijusS) [inline daemon/vpn/nordlynx/libtelio/libtelio.go:160]: This is minor, but it seems to me that both `lana` is dependent on `nurse` and vice-versa. So I'd either write this 
```
if telioConfig.Lana != nil {
<set lana stuff>
} 
if telioConfig.Nurse != nil {
<set nurse stuff>
}
```
or a safer approach:
```
if telioConfig.Lana != nil && telioConfig.Nurse != nil {
 <set lana and nurse stuff>
} else {
 telioConfig.Lana = nil
 telioConfig.Nurse = nil
}
```

**Review by EmilijusS — APPROVED** [review]

File: daemon/vpn/nordlynx/libtelio/libtelio.go (hunk 2 above)
Comment (keliramu) [reply daemon/vpn/nordlynx/libtelio/libtelio.go:160]: In `telio` doc I find only `Nurse` being dependent on `Lana`, but not vice versa.

File: daemon/vpn/nordlynx/libtelio/libtelio.go (hunk 1 above)
Comment (keliramu) [reply daemon/vpn/nordlynx/libtelio/libtelio.go:163]: Yes, you are right. To be removed.

**Pull request merged**